packages/core/lib/search/metaheuristics/RandomSearch.ts

```diff
@@ -106,6 +109,7 @@ export class RandomSearchFactory<T extends Encoding>
       throw new Error("RandomSearch requires runner option.");
     }
     return new RandomSearch(
+      options.eventManager,
```

Comment:
You missed the options check for the search algorithm

Reply:
Fixed